ros2/diagnostics/README.md

```diff
@@ -0,0 +1,7 @@
+# Diagnostics tutorial (C++)
```

Comment:
I'd also mention here that one would normally use the `diagnostics_updater` package (with a link to it on rosindex) and that the way it is done here is only for illustration purposes